static/js/src/advantage/subscribe/react/APICalls/PendingPurchase.jsx

```diff
@@ -0,0 +1,77 @@
+import { useStripe } from "@stripe/react-stripe-js";
```

Comment:
This is a `jsx` file with a capitalized name that's not a component. It appears to be a hook instead. Can you please collect the hooks into a centralized location and use the hook file naming convention of `usePendingPurchase.js`